build/src/lib.rs

```diff
@@ -519,4 +703,76 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn build_with_zstd_transform() -> Result<()> {
+        let tempdir = tempdir::TempDir::new("builder_test")?;
+        let tempdir = tempdir.path();
+
+        let builder = BinaryBuilder::new(&["en"], tempdir)
+            .version("0.3.0")
+            .transform(
+                &|mut buffer, mut writer| {
+                    let mut data = Vec::new();
+                    buffer.read_to_end(&mut data)?;
+                    let data = smush::encode(
+                        data.as_slice(),
+                        smush::Codec::Zstd,
+                        smush::Quality::Maximum,
+                    )?;
+                    writer.write_all(&data)?;
+
+                    // XXX hangs forever, both with `smush` and `brotli` directly
```

Comment:
In my opinion we can remove the "XXX" comments with the broken `brotli` compression.